### PR TITLE
Fixing _guessMass and refactoring vdw module

### PR DESCRIPTION
**Pull request by j3mdamas**
@stefdoerr could you make sure I did not screw voxeldescriptors?
@raimis could you make sure that I did not screw parameterize?

thank you both

### CODE REVIEW

File: htmd/molecule/voxeldescriptors.py [hunk 1]
```diff
@@ -259,9 +259,9 @@ def _getRadii(mol):
             raise ValueError('PDBQT element {} does not exist in mappings.'.format(elem))
         elem = mappings[elem]
 
-        if elem in VDW.elements:
-            rad = radiusByElement(elem)
-        else:
+        try:
```
Comment (stefdoerr) [inline htmd/molecule/voxeldescriptors.py]: no, thats just ugly. why try catch when you can check?

Comment (stefdoerr) [reply htmd/molecule/voxeldescriptors.py]: Or check in the function instead of here

Comment (j3mdamas) [reply htmd/molecule/voxeldescriptors.py]: sorry, stupidity from my side :)

**Comment by j3mdamas** [comment]
@stefdoerr do you approve?

**Pull request merged**